meson_options.txt

```diff
@@ -17,6 +17,7 @@ option('enable-memory-swap', type: 'boolean', value: false)
 option('memory-swap-path', type: 'string', value: '')
 option('test-timeout', type: 'integer', value: 60)
 option('opencl-kernel-path', type: 'string', value: 'nntrainer_opencl_kernels')
+option('enable-mmap', type: 'boolean', value: true)
```

Comment:
This mmap should be enabled for Android. please add enable-mmap in tools/package_android.sh  https://github.com/nnstreamer/nntrainer/blob/main/tools/package_android.sh